### ISSUE TITLE
Small p-values

### ISSUE THREAD

**Issue by jdarias93** [welchr/swiss#10]
Please describe the issue: 

* What is the problem?
SWISS doesn't seem to capture really small p-values (10^-300 and lower). Is there a limit to how small a p-value can be for SWISS to identify it? It seems to ignore these markers.

* Is it reproducible? 
* What was the expected output? 

#### Program output

Output was standard except it didn't include the markers with extremely low p-values

#### Swiss version

If you don't know your swiss version, please do: 1.0.0

```bash
swiss --version
```

**Comment by welchr** [comment]
Following up from email - was it sufficient to change your p-values to log10 scale (including the p-value threshold?) We can fix this via a code update if needed, but would take some time to implement due to other priorities here.

**Comment by jdarias93** [comment]
Dear Dr. Ryan Welch,

Thank you for following up!

Yes I think it worked, but there were slight issues. The first is that it takes a while to take the log of all the p-values. I used the "bc" bash calculator to do it, but there might be a faster way to do it. Also, I didn't know this before, but apparently computers don't automatically pull out an exponent before calculating a log (they just assume a number like 5.3e-200 is 0), so it took some time to work around that. 

Also, I wanted to ask, does SWISS have a way to use a log(pval) threshold (i.e., a negative value)? Since SWISS only accepts p-value threshold between 0 & 1, what I did was I sorted the SWISS output files by log(pval) and just cut out all SNPs after log(pval)>log(5e-9). 

We're also running SWISS with inverse-chi-squared values instead of p-values, which might be a simpler workaround.

Best,
Josh
